Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Ride/EndRide/Internal.hs

```diff
@@ -117,7 +112,6 @@ endRideTransaction ::
   Id Merchant ->
   m ()
 endRideTransaction driverId booking ride mbFareParams mbRiderDetailsId newFareParams thresholdConfig merchantId = do
-  DLoc.updateOnRide merchantId ride.driverId False
```

Comment:
@chakri642 Here also can we check if Ride Cancelled of LTS service is called somewhere in the later flow or previous flow.

Reply:
Handled.
```tripEndPoints <- do
            res <- LF.rideEnd rideId tripEndPoint.lat tripEndPoint.lon booking.providerId driverId
            pure $ toList res.loc```